### PR TITLE
Updated enemies and treacheries for Dim Carcosa encounter cards.

### PR DESCRIPTION
**Pull request by akaan**
Please review my updates on enemies and treacheries for Dim Carcosa.

### CODE REVIEW

**Comment by drawntotheflame** [comment]
Just 3 dots is fine. 
Thank you for this. Will look through it tomorrow or on Saturday and do locations / act / agenda and scenario card.

File: pack/ptc/dca_encounter.json [hunk 1]
```diff
@@ -376,60 +376,73 @@
...
-        "text": "<b>Prey</b> - Most remaining sanity.<br/>Hunter. Massive.<br/>If Hastur's attack deals you horror in excess of your sanity, you take 1 damage, as well.<br/>While Hastur is being attacked or evaded by an investigator with no remaining sanity, treat each [elder_sign], \"+1,\" \"0,\" or \"-1\" token that investigator reveals as a [auto-fail] token.",
+        "text": "<b>Prey</b> - Most remaining sanity.\nHunter. Massive.\nIf Hastur's attack deals you horror in excess of your sanity, you take 1 damage, as well.\nWhile Hastur is being attacked or evaded by an investigator with no remaining sanity, treat each [elder_sign], \"+1,\" \"0,\" or \"-1\" token that investigator reveals as a [auto-fail] token.",
         "traits": "Ancient One. Elite.",
         "type_code": "enemy"
     },
     {
         "code": "03334",
         "encounter_code": "dim_carcosa",
+        "encounter_position": 25,
+        "enemy_damage": 0,
```
Comment (drawntotheflame) [inline pack/ptc/dca_encounter.json]: Don't think you need to add values for 0. Just don't add the line at all - that's been fine previously.

Comment (akaan) [reply pack/ptc/dca_encounter.json]: OK, I removed those lines.

File: pack/ptc/dca_encounter.json [hunk 2]
```diff
@@ -376,60 +376,70 @@
...
         "illustrator": "Stephen Somers",
-        "name": "Hastur",
+        "is_unique": true,
+        "name": "Hastur",
         "pack_code": "dca",
         "position": 334,
         "quantity": 1,
         "subname": "The Tattered King",
-        "text": "<b>Prey</b> - Most remaining sanity.<br/>Hunter.<br/>If Hastur's attack deals you horror in excess of your sanity, you take 1 damage, as well.<br/>While Hastur is being attacked or evaded by an investigator with no remaining sanity, treat each [elder_sign], \"+1,\" \"0,\" or \"-1\" token that investigator reveals as a [auto-fail] token.",
+        "text": "<b>Prey</b> - Most remaining sanity.\nHunter.\nIf Hastur's attack deals you horror in excess of your sanity, you take 1 damage, as well.\nWhile Hastur is being attacked or evaded by an investigator with no remaining sanity, treat each [elder_sign], \"+1,\" \"0,\" or \"-1\" token that investigator reveals as a [auto-fail] token.",
```
Comment (drawntotheflame) [inline pack/ptc/dca_encounter.json]: Please change [auto-fail] to [auto_fail]. (For future reference: the data from cardgamedb uses [per-investigator], [elder-sign] and [auto-fail] and arkhamdb uses [per_investigator], [elder_sign] and [auto_fail].)

File: pack/ptc/dca_encounter.json [hunk 3]
```diff
@@ -376,60 +376,70 @@
...
         "illustrator": "Stephen Somers",
-        "name": "Hastur",
+        "is_unique": true,
+        "name": "Hastur",
         "pack_code": "dca",
         "position": 333,
         "quantity": 1,
         "subname": "Lord of Carcosa",
-        "text": "<b>Prey</b> - Most remaining sanity.<br/>Hunter. Massive.<br/>If Hastur's attack deals you horror in excess of your sanity, you take 1 damage, as well.<br/>While Hastur is being attacked or evaded by an investigator with no remaining sanity, treat each [elder_sign], \"+1,\" \"0,\" or \"-1\" token that investigator reveals as a [auto-fail] token.",
+        "text": "<b>Prey</b> - Most remaining sanity.\nHunter. Massive.\nIf Hastur's attack deals you horror in excess of your sanity, you take 1 damage, as well.\nWhile Hastur is being attacked or evaded by an investigator with no remaining sanity, treat each [elder_sign], \"+1,\" \"0,\" or \"-1\" token that investigator reveals as a [auto-fail] token.",
```
Comment (drawntotheflame) [inline pack/ptc/dca_encounter.json]: Please change [auto-fail] to [auto_fail].

**Comment by drawntotheflame** [comment]
Hi @Kamalisk, I've checked @akaan's work against the cards, and it's all great - just waiting on changing [auto-fail] to [auto_fail] and then this portion is ready to be pulled.

**Comment by akaan** [comment]
It should be all good now.

**Pull request merged**